draft-ietf-rats-corim.md

```diff
@@ -88,20 +88,29 @@ trustworthiness of a remote Attester and therefore to decide whether to engage
 in secure interactions with it. Evidence about trustworthiness can be rather
 complex and it is deemed unrealistic that every Relying Party is capable of the
 appraisal of Evidence. Therefore that burden is typically offloaded to a
-Verifier.  In order to conduct Evidence appraisal, a Verifier requires not only
+Verifier. In order to conduct Evidence appraisal, a Verifier requires not only
 fresh Evidence from an Attester, but also trusted Endorsements and Reference
 Values from Endorsers and Reference Value Providers, such as manufacturers,
-distributors, or device owners.  This document specifies Concise Reference
-Integrity Manifests (CoRIM) that represent Endorsements and Reference Values in
-CBOR format.  Composite devices or systems are represented by a collection of
-Concise Module Identifiers (CoMID) and Concise Software Identifiers (CoSWID)
-bundled in a CoRIM document.
+distributors, or device owners. This document specifies the information elements for
+conveying Endorsements and Reference Values as CBOR, that shall be used by Endorsers
+& Reference Value providers.
 
 --- middle
 
 # Introduction
 
-[^issue] https://github.com/ietf-rats-wg/draft-ietf-rats-corim/issues/4
+Supply chain entities provide the necessary Endorsements and Reference Values refer
+{{Section 4.2 of -rats-arch}} associated with an Attester to a Verifier.
+Some examples of supply chain entites are Original Equipement Manufacturers (OEMs), Original
+Device Manufactueres (ODMs), firmware & software vendors. In a complex
+supply chain, it is likely multiple entities will produce these values at different point in time.
+
+This document specifies Concise Reference Integrity Manifests (CoRIM), a data model that
+contains the Endorsements and Reference Values. The model defined in this document facilitate harmonisation between different supply chain entities.
+
+Inside CoRIM, Concise Module ID (CoMID) tags contain metadata and claims about the hardware and firmware modules. CoMID tags describe platforms, devices and components. A CoSWID Tag represents a Concise representation of a SWID tag and can be implemented in a memory efficient manner. The tag type is extensible so that future revisions can extend it to carry additional information for example, Concise TA Store.
```

Comment:
clarify and streamline

```suggestion
Alongside metadata identifying the authoring supply chain entities, the time validity of the contained information, etc., a CoRIM contains a collection of "tags".

Tags can be of different types:

* Concise Module ID (CoMID) tags contain metadata and claims about hardware and firmware modules;
* CoSWID tags represent software components;
* Concise Bill of Material (CoBOM) contain the list of CoMID and CoSWID that are "active" at a certain point in time.

The set of tags is extensible so that future specifications can add new kinds of information.  For example, Concise Trust Anchor Stores (CoTS) {{?I-D.draft-ietf-rats-concise-ta-stores}} are currently being defined as a standard CoRIM extension.
```